react/src/components/accordions/SprkAccordion.stories.js

```diff
@@ -19,7 +19,7 @@ the [Toggle Component](/docs/components-toggle--default-story) instead.
 };
 
 export const defaultStory = () => (
-  <SprkAccordion>
+  <SprkAccordion additionalClasses="sprk-c-Accordion--no-top-border">
```

Comment:
The issue has it so that this is an option. So we would want to not add it here as the default.

Reply:
We would usually make the component have a new prop (for react) and a new Input (for angular) that would add this class instead of additionalClasses. If that is something that you could update this PR with, that would be better than having this as an additionalClass.